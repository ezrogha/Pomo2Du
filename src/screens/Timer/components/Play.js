import React  from 'react'
import { Polygon } from 'react-native-svg';

export default ({ play }) => (
    <Polygon
        points="180,235 180,285 220,260"
        fill="white"
        stroke="#009DDD"
        strokeWidth="5"
        onPressIn={play}
    />
)
