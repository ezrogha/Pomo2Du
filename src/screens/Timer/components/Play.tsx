import React  from 'react'
import { Polygon } from 'react-native-svg';

type Props = {
    play: () => void
}

export default ({ play }: Props) => (
    <Polygon
        points="180,235 180,285 220,260"
        fill="white"
        stroke="#009DDD"
        strokeWidth="5"
        onPressIn={play}
    />
)
