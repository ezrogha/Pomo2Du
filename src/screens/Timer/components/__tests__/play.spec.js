import React from 'react'

import Play from '../Play';
import renderer from 'react-test-renderer'

describe("Testing Play Component", () => {
    test("renders correctly", () => {
        const tree = renderer.create(<Play />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
