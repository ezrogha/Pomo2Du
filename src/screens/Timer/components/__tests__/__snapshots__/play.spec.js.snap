// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Play Component should render correctly 1`] = `
<RNSVGPath
  d="M180 235 180 285 220 260z"
  fill={
    Array [
      0,
      4294967295,
    ]
  }
  fillOpacity={1}
  fillRule={1}
  matrix={
    Array [
      1,
      0,
      0,
      1,
      0,
      0,
    ]
  }
  opacity={1}
  originX={0}
  originY={0}
  propList={
    Array [
      "fill",
      "stroke",
      "strokeWidth",
    ]
  }
  rotation={0}
  scaleX={1}
  scaleY={1}
  skewX={0}
  skewY={0}
  stroke={
    Array [
      0,
      4278230493,
    ]
  }
  strokeDasharray={null}
  strokeDashoffset={null}
  strokeLinecap={0}
  strokeLinejoin={0}
  strokeMiterlimit={4}
  strokeOpacity={1}
  strokeWidth="5"
  vectorEffect={0}
  x={0}
  y={0}
/>
`;
